napari/layers/_tests/test_layer_actions.py

```diff
@@ -88,3 +89,11 @@ def test_convert_layer(input, type_):
     _convert(ll, type_)
     assert ll[0]._type_string == type_
     assert np.array_equal(ll[0].scale, original_scale)
+    if (
+        type_ == "labels"
+        and isinstance(input, Image)
+        and input.data.dtype == int
+    ):
+        assert id(input.data) == id(
```

Comment:
I also thought about using `numpy.ndarray.base`, but this didn't work the way I expected it to in the test. I think I misunderstood how to use it properly, but it could be another option. https://numpy.org/doc/stable/reference/generated/numpy.ndarray.base.html

Reply:
But the `base` is about object sharing memory, and here we have precisely the same object.